### PR TITLE
Core packs change

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-4534

## Description
Changing of the pre-installed packs on both MPs per the ticket referenced.

### CODE REVIEW

File: Tests/Marketplace/core_packs_list.json [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
+  "EDL",
+  "FeedMitreAttackv2",
+  "FeedTAXII",
+  "FeedUnit42v2",
+  "FiltersAndTransformers",
+  "HelloWorld",
+  "ImageOCR",
+  "Palo_Alto_Networks_WildFire",
+  "Panorama",
+  "Redlock",
```
Comment (yaakovpraisler) [inline Tests/Marketplace/core_packs_list.json:24]: Didn't you mean this?
```suggestion
  "PAN-OS",
  "PrismaCloud",
```

File: Tests/Marketplace/core_packs_list.json [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
-  "CommonTypes",
+  "AutoFocus",
+  "Base",
   "CommonDashboards",
+  "CommonPlaybooks",
   "CommonReports",
+  "CommonScripts",
+  "CommonTypes",
   "CommonWidgets",
+  "CortexXDRIR",
```
Comment (yaakovpraisler) [inline Tests/Marketplace/core_packs_list.json:11]:
```suggestion
  "CortexXDR",
```

File: Tests/Marketplace/core_packs_list.json (hunk 1 above)
Comment (bakatzir) [reply Tests/Marketplace/core_packs_list.json:24]: Should it be the directory name? if so you are correct.

**Comment by yaakovpraisler** [comment]
Triggered [Upload-flow](https://code.pan.run/xsoar/content/-/pipelines/4134385)

**Comment by yaakovpraisler** [comment]
Test upload fails on `Exception: New mandatory dependencies ['AzureSQLManagement', 'AzureNetworkSecurityGroups', 'AzureStorage'] were found in the core pack PrismaCloud`
Screen Shot 2022-12-01 at 17 15 09

@ShahafBenYakir - Adding the new dependencies to the PrismaCloud pack_metadata.json should solve this, WDYT?

**Comment by ShahafBenYakir** [comment]
WDYM? this is the same as we had with ASM, if we want some packs to be added to core we need to make sure they have no required dependencies or add these dependencies to core as well.

**Comment by bakatzir** [comment]
indeed. rming prismacloud pack.

**Comment by yaakovpraisler** [comment]
I'm not really sure why it didn't happend in the previous time but the last test upload failed on: `Exception: New mandatory dependencies ['Campaign', 'PaloAltoNetworks_IoT', 'NIST', 'Phishing', 'Change_Management', 'Identity', 'GoogleChronicleBackstory', 'MajorBreachesInvestigationandResponse', 'SANS'] were found in the core pack CortexXDR`

Anyway, as @ShahafBenYakir said in the above comment, need to check the dependencies why are they mandatory and whether can we add them as not mandatory or maybe not adding `CortexXDR` as core.

**Review by DeanArbel — APPROVED** [review]

**Pull request merged**